start_services_vastai.sh

```
@@ -39,63 +39,35 @@ else
-    if command -v node &> /dev/null && command -v npm &> /dev/null; then
-        FRONTEND_PORT="${FRONTEND_PORT:-13000}"
+    # Start if build now exists
+    if [ -d "frontend/.next" ] && command -v node &> /dev/null; then
         echo "🎨 Starting Next.js frontend on port $FRONTEND_PORT..."
-
-        # Load NVM (if available, for local setup consistency)
-        export NVM_DIR="$HOME/.nvm"
-        # shellcheck disable=SC1091
-        [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-
         cd frontend
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)